web/pages/groups.tsx

```diff
@@ -89,29 +49,22 @@ export default function Groups(props: { groups: SearchGroupInfo[] }) {
               />
             )}
           </Row>
-          <Subtitle>Top Communities</Subtitle>
-          {communities.map((c, i) => (
-            <Community
-              name={c.name}
-              description={c.description}
-              selected={i === selectedCommunity}
-              onClick={() => setSelected(i)}
-              groups={props.groups.filter((g) => c.slugs?.includes(g.slug))}
+          {user && (
+            <UncontrolledTabs
+              className={'mb-4'}
+              tabs={[
+                {
+                  title: 'Your Groups',
+                  content: <YourGroups yourGroupIds={yourGroupIds} />,
+                },
+                {
+                  title: 'Discover',
+                  content: <DiscoverGroups yourGroupIds={yourGroupIds} />,
+                },
```

Comment:
For users that are not in any groups, what do you think about defaulting to the Discover groups tab?

Reply:
ohh smart, will do